.github/workflows/buf-ci.yaml

```diff
@@ -0,0 +1,18 @@
+name: buf
```

Comment:
File renamed for improved support for IDEs as `buf.yaml` can check it implemented a buf configuration file.